github4s/shared/src/test/scala/github4s/integration/GHIssuesSpec.scala

```diff
@@ -36,7 +35,7 @@ trait GHIssuesSpec[T] extends BaseIntegrationSpec[T] {
     })
   }
 
-  "Issues >> Get" should "return an issue which is a PR" in {
+  "Issues >> Get" should "return an issue which is a PR" ignore {
```

Comment:
Should we create an issue for this?

Reply:
I think it's related to https://github.com/47deg/github4s/issues/233